README.md

```diff
@@ -4,21 +4,56 @@ Pop!\_OS is designed for people who use their computer to create; whether it’s
 
 Pop!\_OS’s first release was on October 19th, 2017. For more information, [visit the Pop!\_OS website](https://system76.com/pop) and [view the Pop!\_OS documentation](https://pop.system76.com/docs/).
 
+To view a list of all included repositories, see [REPOS.md](./REPOS.md)
+
 ## Purpose
 
 The purpose of this repository is to allow easy management of all Pop!\_OS related source code and assets.
 
 Binary packages are hosted [on Launchpad](https://launchpad.net/~system76/+archive/ubuntu/pop/+packages). Many packages have source on Github, under the [Pop!\_OS organization](https://github.com/pop-os).
 
-To view a list of all included repositories, see [REPOS.md](./REPOS.md)
+## Developer Resources
+
+For instructions on how to build the shell:
+
+* [COSMIC (GNOME-based)](https://github.com/pop-os/cosmic)
+* [COSMIC-Epoch (Rust-based)](https://github.com/pop-os/cosmic-epoch)
+ 
+Developer chat: https://chat.pop-os.org/pop-os/channels/development
+
+## How to Make a Change
+
+### Find the Correct Repo
+
+Before you make a change, you need to find the relevant repository to make a contribution in. See the 'Developer Resources' section for help finding the correct one. 
+
+### Make An Issue
+
+For large features, it's recommended to start with an issue for discussion if it doesn't already exist. Nobody forces you to get an approval, but you will certainly have a higher chance of getting merged, if the discussion reached a consensus on how the feature should be implemented (if at all).
```

Comment:
> Nobody forces you to get an approval, but you will certainly have a higher chance of getting merged, if the discussion reached a consensus on how the feature should be implemented (if at all).

The last comma in this sentence is grammatically incorrect and should be removed.

This sentence is also confusing and not appropriate for use in documentation-- the word "certainly" is superfluous, and "Nobody forces you to get an approval" is both too informal and inaccurate for the later PR stages. (This is the problem with copy/pasting friendly conversation that people gave you into your work.)

I will replace this sentence myself.

Reply:
Replaced in https://github.com/pop-os/pop/pull/2659/commits/c0c5046cb4a300eb20f441984bf03f9c3ed2f15f.